docs/3_guides/2.2_modify_zkquiz_questions.md

```diff
@@ -0,0 +1,62 @@
+# Modify ZkQuiz Questions
+
+In [Build your first Aligned Application](2_build_your_first_aligned_application.md), we show how to build a trivia application, called ZkQuiz. ZKQuiz asks the user three questions, and if answered correctly, generates a ZK Proof of the correct answers, posts the proof on Aligned, and upon verification, mints an NFT via a smart contract.
+
+In this guide, we will show you how to replace those questions with others.
+
+{% hint style="warning" %}
+This guide assumes you have already read [Build your first Aligned Application](2_build_your_first_aligned_application.md)
+{% endhint %}
+
+## 1. Modify the Questions Asked
+
+First, we need to modify the questions presented to the user. To do this, navigate to our example in `examples/zkquiz/quiz/script/src/main.rs` and change the questions as needed:
+
```

Comment:
I would add a brief disclaimer here to explain that this is a modified version and not the original. Something like: 'Here’s a new set of questions with their respective answers.'

Reply:
Done!